app/models/consumer_app.rb

```diff
@@ -51,9 +51,9 @@ def auth_credentials
   def clear_rpush_app
     case ConsumerApp.platforms[platform_was]
     when Device::IOS
-      Rpush::Apns2::App.where(name: app_bundle_was).first&.destroy
+      Rpush::Apns2::App.where(name: "#{app_bundle_was}.#{platform}").first&.destroy
```

Comment:
Maybe only build this string once before the `case` and then reuse it like you did in `app/queries/consumer_apps/rpush_app_query.rb`.